src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.HillClimbing.Complex.cs

```diff
@@ -7,7 +7,7 @@ internal sealed partial class PortableThreadPool
     {
         private sealed partial class HillClimbing
         {
-            private struct Complex
+            private readonly struct Complex
```

Comment:
Is this a case where moving `System.Numerics.Complex` down would be viable and allow removing duplicated code?

Reply:
Maybe? This is a very limited version of it, doesn't implement interfaces, etc.  If it weren't for possible size issues, I'd say "why not", but I think it'd be worth measuring the impact it has on native aot size, given that the thread pool is used by every app and Complex is used by comparatively few.